controllers/toDos/getToDoById.js

```diff
@@ -0,0 +1,14 @@
+async function getToDoById(req, res, list) {
+    const id = req.params.id;
+
+    for (let toDo of list) {
```

Comment:
anche qui una find con filtro su id magari, ed anche negli altri controllers in generale.